### DISCUSSION TITLE
How to modify the small icon on the left side of CollapsingState

### DISCUSSION THREAD

**Discussion by KUKU-zero2** [emilk/egui#5327] [Q&A] [answered]
As the title suggests, the small icon of CollapsingState matches the background color. How can I change it? Thank you

**Comment by YgorSouza** [comment, marked as answer]
Use [show_toggle_button](https://docs.rs/egui/0.29.1/egui/containers/collapsing_header/struct.CollapsingState.html#method.show_toggle_button) instead of `show_header`. There is an example there in the docs, but if you also want it to toggle when you click on the name, like the default header, you also have to call the `toggle` method that isn't mentioned in the example:

```rs
            let header_res = ui.horizontal(|ui| {
                if ui.label("Header").clicked() {
                    state.toggle(ui);
                }
                state.show_toggle_button(ui, circle_icon);
            });
```

**Reply by KUKU-zero2** [reply]
> 使用[show_toggle_button](https://docs.rs/egui/0.29.1/egui/containers/collapsing_header/struct.CollapsingState.html#method.show_toggle_button)代替`show_header`。文档中有一个示例，但如果您还希望在单击名称时切换它（如默认标题），您还必须调用`toggle`示例中未提及的方法：
> 
> ```rust
>             let header_res = ui.horizontal(|ui| {
>                 if ui.label("Header").clicked() {
>                     state.toggle(ui);
>                 }
>                 state.show_toggle_button(ui, circle_icon);
>             });
> ```

Thank you for your answer. This is indeed a good method, and writing it as a trait is very useful.
